core/lib/spree/testing_support/factories/credit_card_factory.rb

```diff
@@ -7,7 +7,7 @@ def remove_readonly_attributes(attributes) attributes; end
   factory :credit_card, class: TestCard do
     verification_value 123
     month 12
-    year 2013
+    year { Time.now.year }
```

Comment:
I would probably write `Time.now.year + 1` just in case we run the test on new year's eve. ;-)